miasm2/os_dep/linux_stdlib.py

```diff
@@ -36,13 +36,13 @@ def xxx_puts(jitter):
 
     writes the string s and a trailing newline to stdout.
     '''
-    ret_addr, (s,) = jitter.func_args_stdcall(1)
-    while True:
-        c = jitter.vm.get_mem(s, 1)
-        s += 1
-        if c == '\x00':
-            break
-        stdout.write(c)
+    ret_addr, args = jitter.func_args_stdcall(['s'])
```

Comment:
This introduces **one** duplicated line. I like this code better than:

    char = '\xff' # or anything else than '\x00'
    while char != '\x00':
        char = jitter.vm.get_mem(index, 1)
        stdout.write(char)
        index += 1